tests/assets/smb_example_config.yml

```diff
@@ -0,0 +1,26 @@
+root-dir: ~/Documents/HSR/semester_06
+
+global-ignore:
+  - Thumbs.db
+  - .DS_Store
+
+plugins:
+  - name: skripte-server
+    type: smb
+    # hostname: svm-c213.hsr.ch
+    # port: 445
+    # share: skripte
+    # domain: HSR
+    username: example_user
+  # ...
+
+syncs:
```

Comment:
I think we should still come up with a better name for this. Especially in the code, if you don't know about the config file, you have no idea what a `sync` or `syncs` should be. It feels like a word we invented, but doesn't make sense to anyone else.

Ideas:

- Repositories (what openhsr-connect calls it, but it sounds like related to git)
- Modules (as in HSR-Module - bad because of exactly that ambiguity)
- Profiles (I think that's what we called them quite early - with a configuration consisting of many profiles).
- Folders
  - Directories
- Places
  - Locations
- Lessons
  - Courses
  - Subjects
- Documents

Reply:
I would be for "subjects". It's not the best but those are basically a group of all files for one subject. In the US and Canada they call it "course" but I think that is too ambiguous.